README.md

```diff
@@ -1,8 +1,14 @@
 # propagate
 
-[![Build Status](https://travis-ci.org/nock/propagate.svg?branch=master)](https://travis-ci.org/nock/propagate)
+[![npm](https://img.shields.io/npm/v/nock.svg?style=flat-square)][npmjs]
+[![Build Status](https://img.shields.io/travis/nock/propagate/master.svg?style=flat-square)][build]
+[![Coverage](https://img.shields.io/coveralls/github/nock/propagate.svg?style=flat-square)][coverage]
```

Comment:
Coverage badge is incorrectly showing 100% on master. Hopefully that will stabilize…